### ISSUE TITLE
Verschieben-Berechtigung einstellbar machen

### ISSUE THREAD

**Issue by DieGoldeneEnte** [Philipp-g/muesli#7]
?Wollen wir das wirklich?
Es sieht so aus als wäre es möglich das mit akzeptabelen Aufwand zu implementieren. DIe Frage ist, ob das Preis/Leistungs Verhältnis stimmt. Besonders, da dadurch die Komplexität für den Professor steigt. Ich sehe zumindest keine Probleme damit den Tutoren diese Rechte einfach zu geben.

Dem Proffessor die Möglichkeit geben den Tutoren zu erlauben/verbieten (Vorlesungsweit) Studenten zu verschieben.

**Comment by tynsh** [comment]
Nee, das muss nicht unbedingt sein. Kann man weglassen. Allerdings sollte das Verschieben von Studis in andere Übungsgruppen in der Ansicht der Dozenten einfacher gestaltet werden. Z.B. in dem man den Platz von Studis explizit tauschen kann.
